modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,589 @@
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <modules/skybrowser/skybrowsermodule.h>
+
+#include <modules/skybrowser/include/utility.h>
+#include <openspace/engine/moduleengine.h>
+#include <openspace/engine/globals.h>
+#include <openspace/engine/windowdelegate.h>
+#include <openspace/rendering/renderengine.h>
+#include <openspace/scripting/scriptengine.h>
+#include <ghoul/filesystem/filesystem.h>
+#include <ghoul/logging/logmanager.h>
+
+namespace {
+    constexpr const char _loggerCat[] = "SkyBrowserModule";
+} // namespace
+
+namespace openspace::skybrowser::luascriptfunctions {
+
+int selectImage(lua_State* L) {
+    // Load image
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::selectImage");
+    const int i = ghoul::lua::value<int>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    if (module->isCameraInSolarSystem()) {
+        TargetBrowserPair* selected = module->getPair(module->selectedBrowserId());
+        if (selected) {
+            const ImageData& image = module->getWwtDataHandler()->getImage(i);
+            // Load image into browser
+            LINFO("Loading image " + image.name);
+            selected->selectImage(image, i);
+
+            bool isInView = isCoordinateInView(image.equatorialCartesian);
+            // If the coordinate is not in view, rotate camera
+            if (image.hasCelestialCoords && !isInView) {
+                module->startRotatingCamera(
+                    equatorialToGalactic(image.equatorialCartesian)
+                );
+            }
+        }
+    }
+        
+    return 0;
+}
+
+int setHoverCircle(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::setHoverCircle");
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    std::string id = ghoul::lua::value<std::string>(L, 1);
```

Comment:
this can be const? `const std::string id = ghoul::lua::value<std::string>(L, 1);`